app/assets/stylesheets/common/components/admin-post-menu.scss

```diff
@@ -17,5 +21,32 @@
         margin-bottom: 0;
       }
     }
+
+    .btn {
+      justify-content: left;
+      text-align: left;
+      background: none;
+      width: 100%;
+      padding: 0.5em;
+      border-radius: 0;
+      margin: 0;
+
+      .d-icon {
+        color: var(--primary-medium);
+        align-self: flex-start;
+        margin-right: 0.75em;
+        margin-top: 0.1em;
+      }
+
+      &:focus,
+      &:hover {
+        color: var(--primary);
+        background: var(--d-hover);
+
+        .d-icon {
+          color: var(--primary-medium);
+        }
+      }
```

Comment:
do we need to carry this around? What exactly is it adding compared to base styles? if it's super minor, just remove it.

Reply:
It was my idea, but a flat button with a background only on hover/focus is not something we can do with the default button classes, at least, I didn't find a way. Maybe we can add them to the default DButton component in another PR

Reply:
btn-transparent ?

Reply:
I guess it does the trick :+1: